CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 - Added devtool test `-m|--cpuset-mems` flag for memory confinement when tests
   run.
 - Added the virtio traditional memory ballooning device.
+- Added a mechanism for handling fatal errors, which will trigger a process kill when encountered.
```

Comment:
The process is not killed. I would make this more specific - Added a mechanism to handle vCPU/VMM errors that result in process termination.